### PR TITLE
Lock for more Linux Desktop Environments

### PR DESCRIPTION
**Pull request by lockenkop**
I added commands and tested them for:
- cinnamon
- xfce
- plasmax11/wayland
- hyprland
- sway (the sway meta package, atleast in the arch repos, does not come with its lock package swaylock, i'd guess that alot of sway users  use it)

This is somewhat of an aticipation what users are using. I guess if users don't use the standard way of locking for their respective DE they will know they will have to do it with the terminal Entity.

### CODE REVIEW

**Review by richibrics — APPROVED** [review]
Nice, not tested but you said you tested them so okay

File: IoTuring/Entity/Deployments/Lock/Lock.py [hunk 1]
```diff
@@ -13,10 +13,16 @@
         'base': 'pmset displaysleepnow'
     },
     OsD.LINUX: {
-        'gnome': 'gnome-screensaver-command -l',
+        # 'gnome': 'gnome-screensaver-command -l', as of my knowledge, gnome-screensaver is deprecated
```
Comment (infeeeee) [inline IoTuring/Entity/Deployments/Lock/Lock.py:16]: Yes, but it's still available in a lot of distros: https://pkgs.org/download/gnome-screensaver

Latest debian and Ubuntu LTS still have it.

File: IoTuring/Warehouse/Deployments/HomeAssistantWarehouse/entities.yaml [hunk 2]
```diff
@@ -124,3 +124,8 @@ TerminalButton:
   name: Terminal Button
   icon: mdi:console
   custom_type: button
+Brightness:
```
Comment (infeeeee) [inline IoTuring/Warehouse/Deployments/HomeAssistantWarehouse/entities.yaml:127]: You left this here from an other PR.

**Review by infeeeee — CHANGES_REQUESTED** [review]
Solve the 2 comments, otherwise it's nice.

File: IoTuring/Entity/Deployments/Lock/Lock.py (hunk 1 above)
Comment (infeeeee) [reply IoTuring/Entity/Deployments/Lock/Lock.py:16]: I'm ok with dropping support for gnome-screensaver, but than fully remove it, not just comment. We can get it back from git history anyway.

Comment (lockenkop) [reply IoTuring/Entity/Deployments/Lock/Lock.py]: I added it again, since i think dropping support for debian and Ubuntu LTS has such a big userbase.

**Review by infeeeee — APPROVED** [review]

**Pull request merged**